hack/manifests/base/deployment.yaml

```diff
@@ -43,12 +43,19 @@ spec:
       labels:
         app: workflow-service
     spec:
+      volumes:
+        - name: user-jar
+          persistentVolumeClaim:
+            claimName: user-jar
```

Comment:
let's rename it to `workflows-jar` or `user-workflows-jar`

Reply:
fixed